### DISCUSSION TITLE
CustomTreeItem

### DISCUSSION THREAD

**Discussion by Sergey-Shapovalov** [UI5/webcomponents#6052] [Ideas]
Hello,

Thanks for the useful set of web components.

Is there a plan to implement custom tree items, similar to sap.m.CustomTreeItem or ui5-li-custom?

**Comment by Lukas742** [comment]
Hi @Sergey-Shapovalov 

There's already an open issue covering this:
https://github.com/SAP/ui5-webcomponents/issues/5921

**Comment by nnaydenow** [comment]
Hi @Sergey-Shapovalov,

I'm closing the discussion since the custom tree item component was provided.
